editor/src/app/inputs/text-input.component.ts

```diff
@@ -24,6 +24,13 @@ import { TextInputService } from './text-input.service';
                  (focus)="onFocus()"
                  (keydown)="onKeyDown($event)"
                  (blur)="onBlur($event)">
+          <svg *ngIf="allowBlank && value"
```

Comment:
let's improve the usability of our HTML by telling browser that this is a button.
- add an aria label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Basic_buttons
- and the button role - `role="button"`
- and a `tabindex="0"` so it can be focused